ports/descent3/Descent 3.sh

```diff
@@ -57,7 +56,7 @@ else
 fi
 
 if [ "$LIBGL_FB" != "" ]; then
-  export SDL_VIDEO_GL_DRIVER="$GAMEDIR/gl4es.aarch64/libGL.so.1"
```

Comment:
What does this change have to do with bind mounts?

Reply:
This was one of the issues we had to address because it prevented the port from starting. So, it was done in the context of the bind-directories fixes. I figured not to make 2 PRs for the change.

Reply:
I know but the title of the PR is replace symbolic links and then does other things :P